pysatSpaceWeather/instruments/sw_ace.py

```diff
@@ -0,0 +1,463 @@
+# -*- coding: utf-8 -*-
+"""Supports ACE Magnetometer data
+
+Properties
+----------
+platform
+    'sw'
+name
+    'ace'
+tag
+    - 'rt' Real-time data from the Space Weather Prediction Center (SWPC)
```

Comment:
```suggestion
    - 'real-time' Real-time data from the Space Weather Prediction Center (SWPC)
```

Reply:
Changes to unhyphenated version throughout file in last commit.